test/e2e/addon_update.go

```diff
@@ -210,13 +210,25 @@ var _ = KubeDescribe("Addon update", func() {
 		// Reduce the addon update intervals so that we have faster response
 		// to changes in the addon directory.
 		// do not use "service" command because it clears the environment variables
-		sshExecAndVerify(sshClient, "sudo TEST_ADDON_CHECK_INTERVAL_SEC=1 /etc/init.d/kube-addons restart")
+		if testContext.OSDistro == "debian" {
```

Comment:
maybe use a `switch` instead of this `if-else` tree? it's the pattern we follow for `testContext.Provider`.